pyOCD/gdbserver/gdbserver.py

```diff
@@ -332,21 +466,20 @@ def resume(self):
         self.timeOfLastPacket = time()
```

Comment:
This can be changed to a local variable.  Not directly related to your patch, but figured I would mention it.